src/configuration/configuration.ts

```diff
@@ -470,16 +472,6 @@ class Configuration implements IConfiguration {
   visualModeKeyBindingsMap: Map<string, IKeyRemapping> = new Map();
   commandLineModeKeyBindingsMap: Map<string, IKeyRemapping> = new Map();
 
-  get textwidth(): number {
```

Comment:
This is admittedly a bit janky (config system is in need of an overhaul), but it is necessary to let `textwidth` be configured per-language (see #7747).

I think we should instead add `set textwidth(number)`